### PR TITLE
Fix logo in documentation website

### PR DESCRIPTION
**Pull request by rgaiacs**
Before

Site Reliability Guide for mybinder org before

After

Site Reliability Guide for mybinder org after

BinderHub documentation (https://binderhub.readthedocs.io/en/latest/)

Screenshot 2023-06-14 at 09-38-36 BinderHub

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -16,7 +16,6 @@
 extensions = [
     "myst_parser",
     "sphinx_copybutton",
-    "jupyterhub_sphinx_theme",
```
Comment (manics) [inline docs/source/conf.py:19]: jupyterhub_sphinx_theme is a Sphinx/readthedocs theme we created for the JupyterHub org with some useful defaults. Is there a particular problem you noticed with it, or is it that you think pydata_sphinx_theme looks better?

File: docs/source/conf.py [hunk 2]
```diff
@@ -39,27 +38,36 @@
...
+            "type": "fontawesome",
         },
     ],
-    "use_edit_page_button": True,
 }
 html_context = {
     "github_user": "jupyterhub",
     "github_repo": "mybinder.org-deploy",
-    "github_version": "HEAD",
+    "github_version": "main",
```
Comment (manics) [inline docs/source/conf.py:64]:
```suggestion
    # Branch name is needed to ensure the `Edit on GitHub` button works
    "github_version": "main",
```
Added a comment so this doesn't get reverted in future

File: docs/source/conf.py (hunk 1 above)
Comment (rgaiacs) [reply docs/source/conf.py:19]: The logo information at https://github.com/jupyterhub/mybinder.org-deploy/blob/c1212504cefe965a3ef9acb2a9bafe0e28027320/docs/source/conf.py#L61 is not being ignored by `jupyterhub_sphinx_theme`. https://binderhub.readthedocs.io/ and https://mybinder.readthedocs.io/en/latest/ uses the `pydata_sphinx_theme`. Trying to make the documentation across Binder uniform.

File: docs/source/conf.py [hunk 3]
```diff
@@ -16,7 +16,6 @@
 extensions = [
     "myst_parser",
     "sphinx_copybutton",
```
Comment (consideRatio) [inline docs/source/conf.py:18]:
```suggestion
    "sphinx_copybutton",
    "jupyterhub_sphinx_theme",
```

Let's add this back, the bug of not respecting `html_logo` should be fixed instead, as proposed via https://github.com/jupyterhub/jupyterhub-sphinx-theme/pull/13

File: docs/source/conf.py [hunk 4]
```diff
@@ -39,27 +38,37 @@
 # -- Options for HTML output ----------------------------------------------
 # ref: http://www.sphinx-doc.org/en/master/usage/configuration.html#options-for-html-output
 #
-html_theme = "jupyterhub_sphinx_theme"
+html_theme = "pydata_sphinx_theme"
```
Comment (consideRatio) [inline docs/source/conf.py:41]:
```suggestion
html_theme = "jupyterhub_sphinx_theme"
```

**Review by consideRatio — COMMENTED** [review]
With a version released of [jupyterhub-sphix-theme newer than 0.0.3](https://pypi.org/project/jupyterhub-sphinx-theme/), and code suggestions applied, this is good to go I think! Thank you @rgaiacs!!

**Pull request merged**